logging/logging.go

```diff
@@ -454,3 +470,19 @@ func createLogString(entry *sdjournal.JournalEntry, addUnit bool) (logStr string
 func getLogDate(entry *sdjournal.JournalEntry) (date time.Time) {
 	return time.Unix(int64(entry.RealtimeTimestamp/1000000), int64((entry.RealtimeTimestamp%1000000))*1000)
 }
+
+func getServiceNameFromLog(logEntry *sdjournal.JournalEntry) (serviceName string) {
+	systemdCgroup := logEntry.Fields[sdjournal.SD_JOURNAL_FIELD_SYSTEMD_CGROUP]
```

Comment:
Add empty line befor if. The rule is: select new var definition with empty line.

Reply:
ok. done